package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/user-event": "^12.1.6",
     "@types/yup": "^0.29.8",
     "axios": "^0.20.0",
+    "clsx": "^1.1.1",
```

Comment:
we already have this functionality with `obj-str` https://github.com/lukeed/obj-str

Reply:
fixed 👍